write_ffcv_datasets.py

```diff
@@ -72,6 +77,14 @@
         path = train_beton_fpath if name == "train" else test_beton_fpath
         writer = DatasetWriter(path, {"image": RGBImageField(), "label": IntField()})
         writer.from_indexed_dataset(ds)
+    if dataset=='stl10':
+        datasets = {"unlabeled": unlabeledset}
+        unlabeled_beton_fpath = os.path.join(ffcv_folder, "unlabeled.beton")
+        for name, ds in datasets.items():
+            breakpoint()
```

Comment:
remove `breakpoint()`

Reply:
Kept it to have control before the files are written. 
In case we run it just to verify stats without setting write flag to False, this is where it stops and doesn't overwrite the existing files.